rss_parser/_parser.py

```diff
@@ -19,10 +20,10 @@ def get_soup(xml: str, parser: str = "xml") -> BeautifulSoup:
 
     @staticmethod
     def check_none(
-        item: object,
-        default: str,
-        item_dict: Optional[str] = None,
-        default_dict: Optional[str] = None,
+            item: object,
```

Comment:
I think black uses 4 spaces instead of 8, let's try to not cause any extra diffs

Reply:
Oh I did not, see that I put more spaces, I will fix this in the next commit.